concepts/05 UI Components/NumberBox/00 Getting Started with NumberBox/20 Handle Value Change.md

```diff
@@ -0,0 +1,117 @@
+Users can process a value after they enter it or after they click the spin buttons. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#onValueChanged) callback to handle value changes.
+
+The following code displays a notification every time users change the NumberBox value.
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#number-box").dxNumberBox({
+            // ...
+            onValueChanged(e) {
+                const currentNumber = e.value;
+                if (currentNumber) {
+                    DevExpress.ui.notify({
+                        message: "The sum is $" + currentNumber
+                    });
+                }
+            }
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-number-box ...
+        (onValueChanged) = "onValueChanged($event)"
+    >
+    </dx-number-box>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from "@angular/core";
+    import notify from "devextreme/ui/notify";
+
+    @Component({
+        selector: "app-root",
+        templateUrl: "./app.component.html",
+        styleUrls: ["./app.component.css"]
+    })
+    export class AppComponent {
+        // ...
+        currentNumber: string;
+        onValueChanged(e) {
+            this.currentNumber = e.value;
+            if (this.currentNumber) {
+                notify({
+                    message: "The sum is $" + this.currentNumber
+                });
+            }
+        }
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxNumberBox ...
+            @value-changed = "onValueChanged($event)"
+        />
+    </template>
+
+    <script>
+    // ...
+    import { DxNumberBox } from "devextreme-vue/number-box";
+    import notify from "devextreme/ui/notify";
+
+    export default {
+        components: {
+            DxNumberBox
+        },
+        data() {
+            return {
+                currentNumber: ""
+            };
+        },
+        methods: {
+            onValueChanged(e) {
+                this.currentNumber = e.value;
```

Comment:
You can two-way bind the NumberBox value to `currentNumber` and only use `e.value` to display notifications.

Reply:
Ok